src/docs/product/discover-queries/query-builder.mdx

```diff
@@ -58,6 +58,13 @@ Click on any of these sections to further refine your search. For example, click
 
 ![Facet map for transaction and handled tags. Map looks like a bar with gradient colors.](./discover-facet-map.png)
 
+<Note>
+  When performing searches on transaction events, based on your search criteria
+  and sample rate, the events available may be limited because Discover uses
+  sampled data only. Learn more by reading about [retention
```

Comment:
```suggestion
Discover only uses sampled data for transactions. This means that when performing searches on transaction events, due to your search criteria and sample rate, you might see limited results. Learn more by reading about ...
```
Is it true that Discover only uses sampled data? Are there cases where it uses all data?

Reply:
For transactions it's always sampled, but for low volumes we happen to sample 100% 😅 That's clarified further in the docs about retention priorities, so we can also rephrase it to how you suggested 👍